static/js/map_init.js

```diff
@@ -1,35 +1,71 @@
 function main_map_init (map, options) {
     // Add GeoJSON layer   
-    var marker, org_id;
+    var marker, org_id, org_row, orgs_set, org_rows;
 
     // Add building markers with popups to buildings.
```

Comment:
The idea is to create marker with polygon's centroid coordinates, than create a list (<ul>), which will contain <a> elements with binded select_building function, and insert this list in marker popup. So user click on marker, meet with popup, click on one of the <a> which refers to organizations, and get claims